platform/src/main/resources/example.anchor-config.yaml

```
@@ -95,17 +95,20 @@ metrics:
   run_interval: 30
 
 events:
-  enabled: true
-  publisher_type: kafka
-  options:
-    bootstrap_server: localhost:29092
-    use_single_queue: false
-    event_type_to_queue:
-      all: ap_event_single_queue
-      quote_created: ap_event_quote_created
-      transaction_created: ap_event_transaction_created
-      transaction_status_changed: ap_event_transaction_status_changed
-      transaction_error: ap_event_transaction_error
+  enabled: false
```

We want the example implementation to have the queue enabled, right?

```suggestion
  enabled: true
```

---

Changed to `true`.